superset-frontend/src/components/Button/button.stories.jsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, boolean, select, text } from '@storybook/addon-knobs';
+import Button from './index';
+
+export default {
+  title: 'Button',
+  component: Button,
+  decorators: [withKnobs],
+};
+
+const bsStyleKnob = {
+  label: 'Types',
+  options: {
+    Primary: 'primary',
+    Secondary: 'secondary',
+    Danger: 'danger',
+    Warning: 'warning',
+    Success: 'success',
+    Link: 'link',
+    Default: 'default',
+    None: null,
+  },
+  defaultValue: null,
+  // groupId: 'ButtonType',
+};
+const bsSizeKnob = {
+  label: 'Sizes',
+  options: {
+    XS: 'xsmall',
+    S: 'small',
+    M: 'medium',
+    L: 'large',
+    None: null,
+  },
+  defaultValue: null,
+};
+// TODO remove the use of these in the codebase where they're not necessary
```

Comment:
Leaving this here for the moment... it's the result of taking inventory, and I'll remove these comments as I (at least try to) remove these classes from the codebase